sr_robot_launch/launch/sr_ur10arm_box.launch

```diff
@@ -21,36 +21,41 @@
   <arg name="world" default="$(find sr_description_common)/worlds/demo_space_large.world" if="$(arg scene)"/>
   <!-- Set to true if you want the robot to start in the "home" positions -->
   <arg name="start_home" default="false"/>
+
+  <!-- ROBOT CONFIG-->
+  <arg name="side" default="right"/>
+  <arg name="side_letter" value="$(eval side[0])"/>
+  <arg name="arm_model" default="ur10"/>
+  <arg name="robot_description" default="'$(find sr_box_ur10_moveit_config)/config/$(arg side_letter)a_$(arg arm_model)_with_box.urdf.xacro'"/>
+  <arg name="robot_config_file" default="$(find sr_box_ur10_moveit_config)/config/robot_configs/$(arg side)_$(arg arm_model).yaml"/>
+
   <!-- Definition of "home" position, in joint-angle pairs -->
-  <arg name="home_angles" default="-J ra_shoulder_pan_joint 1.147 -J ra_elbow_joint 1.695 -J ra_wrist_1_joint -1.395 -J ra_wrist_2_joint -1.584 -J ra_shoulder_lift_joint -1.926 -J ra_wrist_3_joint 1.830"/>
+  <arg name="home_angles" unless="$(eval side=='left')" default="-J ra_shoulder_pan_joint 1.147 -J ra_elbow_joint 1.695 -J ra_wrist_1_joint -1.395 -J ra_wrist_2_joint -1.584 -J ra_shoulder_lift_joint -1.926 -J ra_wrist_3_joint 1.830"/>
+  <arg name="home_angles" if="$(eval side=='left')" default="-J la_shoulder_pan_joint 1.147 -J la_elbow_joint -1.695 -J la_wrist_1_joint -1.75 -J la_wrist_2_joint 1.57 -J la_shoulder_lift_joint -1.22 -J la_wrist_3_joint -1.830"/>
   <!-- Set this to true if you run the hardware loop on a remote computer -->
   <arg name="external_control_loop" default="false"/>
   <!-- The robot state publisher publishing frequency  -->
-  <arg name="robot_state_pub_frequency" default="50"/>
+  <arg name="robot_state_pub_frequency" default="250"/>
   <!-- The joint state publisher publishing frequency  -->
-  <arg name="joint_state_pub_frequency" default="100"/>
+  <arg name="joint_state_pub_frequency" default="125"/>
 
   <!-- HAND CONFIG-->
   <arg name="hand_id" default="H1"/>
 
   <!-- ARM CONFIG-->
-  <arg name="arm_id" default="ra"/>
-  <arg name="side" default="right"/>
-  <arg name="arm_model" default="ur10"/>
-  <arg name="initial_z" default="0.7751" if="$(arg scene)"/>
+  <arg name="arm_id" default="$(arg side_letter)a"/>
+  <arg name="initial_z" default="0.7551" if="$(arg scene)"/>
   <arg name="initial_z" default="0.1" unless="$(arg scene)"/>
   <arg name="arm_robot_hw" default="$(find sr_robot_launch)/config/$(arg side)_ur_arm_robot_hw.yaml"/>
-  <arg name="arm_controller" default="$(find sr_robot_launch)/config/$(arg side)_ur_arm_controller.yaml"/>
```

Comment:
Why is this not needed anymore?

Reply:
It seems it was never used in the first place, no other mention of `arm_controller` in the file.